packages/datadog_grpc_interceptor/lib/src/datadog_grpc_interceptor.dart

```diff
@@ -0,0 +1,102 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+import 'dart:io';
+
+import 'package:datadog_flutter_plugin/datadog_flutter_plugin.dart';
+import 'package:datadog_grpc_interceptor/src/tracing_util.dart';
+import 'package:grpc/grpc.dart';
+import 'package:uuid/uuid.dart';
+
+class DatadogTracingHeaders {
+  static const traceId = 'x-datadog-trace-id';
+  static const parentId = 'x-datadog-parent-id';
+
+  static const sampled = 'x-datadog-sampled';
```

Comment:
I guess we can remove this one as the outcome in the APM discussion

Reply:
A yes, thanks!